tests/test_sentinel2_processing_commands.py

```diff
@@ -28,6 +28,7 @@
 from actinia_core.core.common.config import global_config
 import os
 import unittest
+import pytest
```

Comment:
`pytest` is not used?

Reply:
Thanks, now it is due to the unittest decorator.